### PR TITLE
[WIP] Add CRC customization after start

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

**Review by bogdando — COMMENTED** [review]
lgtm

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

File: ci_framework/roles/rhol_crc/files/customizations/patches.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#spec:
+#  unsupportedConfigOverrides:
+#    useUnsupportedUnsafeNonHANonProductionUnstableEtcd: true
+---
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Removed
```
Comment (bogdando) [inline ci_framework/roles/rhol_crc/files/customizations/patches.yml:23]: good catch. That is a new way of turning it off that works indeed

**Comment by bogdando** [comment]
How's going with this? Another related optimization https://access.redhat.com/solutions/4367311
We might want to define storage ResourceQuota for namespaces like openstack openstack-operators at least, ideally also for openshift-marketplace, metallb-system

**Comment by bogdando** [comment]
@cjeanner FYI

**Comment by pablintino** [comment]
Not relevant anymore.

**Pull request closed without merging**